examples/00_getting_started.py

```diff
@@ -57,25 +56,23 @@
                    'project'       : config[resource]['project'],
                    'queue'         : config[resource]['queue'],
                    'access_schema' : config[resource]['schema'],
-                   'cores'         : config[resource]['cores'],
+                   'cores'         : 128
                   }
         pdesc = rp.ComputePilotDescription(pd_init)
 
         # Launch the pilot.
         pilot = pmgr.submit_pilots(pdesc)
 
-        report.header('submit units')
+        n = 1024  # number of units to run
```

Comment:
the same here

Reply:
This was always hard coded, and I'd like to leave that for now, to keep the examples simple (only resource details are in the config right now).

Reply:
OK, works for me.